src/components/coin-row/TopMoverCoinRow.js

```diff
@@ -84,9 +85,11 @@ const TopMoverCoinRow = asset => {
 
   return (
     <ButtonPressAnimation
-      onCancel={({ nativeEvent: { state, close } }) =>
-        state === 5 && close && handlePress()
-      }
+      hapticType="notificationWarning"
+      onCancel={({ nativeEvent: { state, close } }) => {
+        state === 5 && close && handlePress();
+        ReactNativeHapticFeedback.trigger('selection');
```

Comment:
Just wondering if this should be called only when handlePress is triggered (inside handlePress)

Reply:
yes! thank you!